HomeWork/src/main/java/HomeWork5/HomeWork5.java

```diff
@@ -55,41 +54,47 @@ public static void guessTheWordGame () {
 
     public static String stringReverse (String source) {
         if (source == null) {
-            return "Ошибка!";
+                throw new NullPointerException ("Error: you didn't enter any data." + "\n" +
+                        "null was passed to the argument 'source'.");
         } else {
             return new StringBuilder(source).reverse().toString();
         }
     }
     public static int findSymbolOccurance (String name, char symbol) {
 
+        int counter = 0;
         if (name == null) {
-            return 0;
+            Objects.requireNonNull(name, "Error: you didn't enter any data." + "\n" +
+                    "Null was passed to the argument 'name'.");
         } else {
             char[] symbols = name.toCharArray();
-            int counter = 0;
 
             for (int i = 0; i < symbols.length; i++) {
                 if (symbol == symbols[i]) {
                     counter++;
                 }
             }
-            return counter;
         }
+        return counter;
     }
 
     public static int findWordPosition (String source, String target) {
-        if (source == null && target == null) {
-            return -1;
+        if (source == null || target == null) {
+            throw new NullPointerException("Error: you didn't enter any data." + "\n" +
+                    "Null was passed to the argument. class HomeWork5/ line_84");
         } else {
```

Comment:
You have an `if` statement where your validation is located, that's why if validation is passed your code automatically will be executed line by line. So, the else statement is redundant.